src/main/java/core/basesyntax/service/operationhandler/BalanceHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operationhandler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceHandler implements OperationHandler {
+    @Override
+    public void handleOperation(FruitTransaction transaction, Storage storage) {
+        Storage.fruits.put(transaction.getFruit(), transaction.getQuantity());
+    }
```

Comment:
You use static component of Storage class, there is no need to pass object
```suggestion
    public void handleOperation(FruitTransaction transaction) {
        Storage.fruits.put(transaction.getFruit(), transaction.getQuantity());
    }
```